presto-hive/src/main/java/io/prestosql/plugin/hive/security/StaticAccessControlMetadataFactory.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.security;
+
+import io.prestosql.plugin.hive.metastore.SemiTransactionalHiveMetastore;
+
+public class StaticAccessControlMetadataFactory
```

Comment:
Let's put this inside `StaticAccessControlMetadataModule` so that it's consistent with `SqlStandardAccessControlMetadataFactory`.

Reply:
I inlined that.